src/main/java/com/stories/services/StoryServiceImpl.java

```diff
@@ -0,0 +1,36 @@
+package com.stories.services;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.HttpClientErrorException;
+
+import com.stories.domain.StoryDomain;
+import com.stories.model.StoryModel;
+import com.stories.repository.StoriesRepository;
+
+import ma.glasnost.orika.MapperFacade;
+import ma.glasnost.orika.MapperFactory;
+import ma.glasnost.orika.impl.DefaultMapperFactory;
+
+@Service
+public class StoryServiceImpl {
+
+	@Autowired
+	StoriesRepository storiesRepository;
+
+	public StoryDomain updateStory(StoryDomain request, String id) {
+		if (!storiesRepository.existsById(id))
+			throw new HttpClientErrorException(HttpStatus.NOT_FOUND, "Story not found");
+		MapperFactory mapperFactory = new DefaultMapperFactory.Builder().build();
```

Comment:
Separate Mapper logic in a different class and inject the mapper factory bean.
See example [here](https://github.com/IvanVegaS/DnData/blob/develop/src/main/java/com/ivcoding/services/dndata/service/RacesServiceImpl.java)